kodilla-patterns/src/test/java/com/kodilla/patterns/prototype/library/LibraryTestSuite.java

```diff
@@ -29,13 +29,11 @@ public void shouldReturn3booksInShallowCopyTest() {
             cloneLibrary = library.shallowCopy();
             cloneLibrary.setName("Harry Potter in Ostrow WLKP");
         } catch (CloneNotSupportedException e) {
-            System.out.println(e);
+            e.getMessage();
```

Comment:
To zwróci Ci po prostu stringa, ale nie wypisze go nigdzie :) w takim wypadku jak tu lepiej dodać do metody/testu informacje, że może ona rzucić wyjątek, bo nie musisz tego handlować